lib/registration.pm

```diff
@@ -934,7 +934,7 @@ sub scc_deregistration {
         # Remove registercloudguest when use suseconnect in a non public cloud environment.
         if (get_var('SCC_ADDONS', '') =~ /pcm/) {
             my $cloudguest_bin = "/usr/sbin/registercloudguest";
-            assert_script_run "test -f $cloudguest_bin && rm -f $cloudguest_bin";
+            assert_script_run "[ -f $cloudguest_bin ] && rm -f $cloudguest_bin || true";
```

Comment:
It is better to use script_run while not assert, return true is hacky for not found the binary.

Reply:
Changed.